src/test/java/org/kpmp/participant/ParticipantServiceTest.java

```diff
@@ -0,0 +1,67 @@
+package org.kpmp.participant;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+public class ParticipantServiceTest {
+
+	private ParticipantService participantService;
+	@Mock
+	private ParticipantSummaryDatasetRepository participantSummaryDatasetRepository;
+
+	@Before
+	public void setUp() throws Exception {
+		MockitoAnnotations.openMocks(this);
+		participantService = new ParticipantService(
+			participantSummaryDatasetRepository);
+	}
+
+	@After
+	public void tearDown() throws Exception {
+		MockitoAnnotations.openMocks(this).close();
+		participantService = null;
+	}
+
+
+	@Test
+	public void testGetParticipantSummaryDataset() throws Exception {
+
+		ParticipantSummaryDataset expectedResult = new ParticipantSummaryDataset();

```

Comment:
This is perfectly legit, but it would also be ok to just create a mock ParticipantSummaryDataset and make sure it returns the same object. Since this object does not have an equals method, it will only be equal if it is the same object.
No need to change this, just wanted to give you an fyi.

Reply:
I was originally trying something like this, but I was hitting issues with the id (of the two instantiated objects) being different.. Probably not doing something right on my end.